assets/src/domain/eventEditor/ui/ticketAssignmentsManager/useAssignmentManager.ts

```diff
@@ -76,17 +77,21 @@ const useAssignmentManager = (): AM => {
 		}
 	};
 
-	const initialize: AM['initialize'] = (data: TAMRelationalData) => {
+	const initialize: AM['initialize'] = (data: TAMRelationalData, forDate) => {
 		const relationsToPick: Array<keyof TAMRelationalData> = ['datetimes', 'tickets'];
 		// pick only datetimes and tickets from relational data
 		let newData = pick(relationsToPick, data);
 
 		// Remove other relations from newData
 		// like ticket to price relations
-		newData = map((relationalEntity) => {
+		newData = mapObjIndexed((relationalEntity, entity) => {
```

Comment:
nice, didn't used yet `mapObjIndexed` :+1:

Reply:
`mapObjIndexed` is useful when you need the key as well.